ask-sdk-jinja-renderer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ask-sdk-core>=1.1.0
```

Comment:
Update the sdk minimum version here as well

Reply:
Ok.